src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/experimental_feature.yml

```diff
@@ -0,0 +1,15 @@
+admin_experimental_feature_index:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\ExperimentalFeature:index'
+        _legacy_controller: AdminExperimentalFeature
+        _legacy_link: AdminExperimentalFeature
+
+admin_experimental_feature_save:
```

Comment:
I'm not sure about this. I think it's better to separate the needs: one route for fetching, one route for saving. But the same controller is hit.